packages/e2e-tests/specs/editor/various/datepicker.test.js

```diff
@@ -15,12 +15,24 @@ async function getSelectedOptionLabel( selector ) {
 }
 
 async function getDatePickerValues() {
-	const year = await getInputValue( '[aria-label="Year"]' );
-	const month = await getInputValue( '[aria-label="Month"]' );
-	const monthLabel = await getSelectedOptionLabel( '[aria-label="Month"]' );
-	const day = await getInputValue( '[aria-label="Day"]' );
-	const hours = await getInputValue( '[aria-label="Hours"]' );
-	const minutes = await getInputValue( '[aria-label="Minutes"]' );
```

Comment:
Just wondering if it wouldn't be a good thing to add these `aria-label`s in our code (unless those inputs are already labelled in an accessible way)

Reply:
The inputs are labelled, but it's done using `<label for="">` which you can't target using a CSS selector. I thought about using `puppeteer-testing-library` here so that we can do this properly but I think it's too great of a change for this PR.

Reply:
Agreed!